### PR TITLE
[Improvement][Seatunnel-web] Add support to execute seatunnel-web REST API e2e

### PR DESCRIPTION
**Pull request by arshadmohammad**
Added to support run test cases in live seatunnel and seatunnel-web. Please go through seatunnel-web-it/README.md to know how to run these integration tests.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by arshadmohammad** [comment]
Added 31 test cases also. All are passing
image

**Comment by hailin0** [comment]
Great idea

File: seatunnel-web-it/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Build seatunnel-web
+./mvnw clean install -DskipTests
+
+Update mysql database details in src/test/resources/application.yml and Run the seatunnel-web-it integration tests
```
Comment (Hisoka-X) [inline seatunnel-web-it/README.md:4]: How about add mysql environment in e2e? We have some example in https://github.com/apache/seatunnel/blob/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/MysqlCDCIT.java#L107

Comment (arshadmohammad) [reply seatunnel-web-it/README.md:4]: Good suggestion. 
Sure, we can add mysql environment in e2e test cases, but later and in separate PR.

**Comment by Hisoka-X** [comment]
Thanks @arshadmohammad

**Comment by arshadmohammad** [comment]
Thanks @davidzollo and @hailin0  for the reviewing the PR.
Thanks @Hisoka-X for reviewing and merging the PR.

**Pull request merged**